apps/qubit/modules/informationobject/templates/multiFileUploadSuccess.php

```diff
@@ -104,42 +95,48 @@
 <?php end_slot() ?>
 
 <?php slot('after-content') ?>
-  <?php echo javascript_tag(<<<content
-// If JavaScript and Flash Player installed
-if (0 == YAHOO.deconcept.SWFObjectUtil.getPlayerVersion().major)
-{
-  var noflash = jQuery('#no-flash');
-  noflash.show();
-}
-else
-{
-  jQuery('form#multiFileUploadForm').show();
-}
-
-// Uncomment following line to enable logging to Firebug or Safari js console
-//YAHOO.widget.Logger.enableBrowserConsole();
-
-YAHOO.widget.Uploader.SWFURL = '$uploadSwfPath';
-
-Qubit.multiFileUpload.maxUploadSize = '$maxUploadSize';
-Qubit.multiFileUpload.uploadTmpDir = '$uploadTmpDir';
+
+<?php echo javascript_tag(<<<content
+Qubit.multiFileUpload.maxFileSize = '$maxFileSize';
+Qubit.multiFileUpload.maxPostSize = '$maxPostSize';
 Qubit.multiFileUpload.uploadResponsePath = '$uploadResponsePath';
 Qubit.multiFileUpload.objectId = '$resource->id';
```

Comment:
I'd prefer to use the slug, if possible.

Reply:
Using slug instead of object id now for image upload from Uppy to digitalobject/uploadAction.php. See latest commits.